frontend/public/locales/en/translation.json

```diff
@@ -2849,6 +2854,9 @@
   "validate.privateSshKey": "Must be a valid private key",
   "validate.projectID.format": "Must consist of 6 to 30 lowercase alphanumeric characters or '-', and must start with a letter and not end with '-'",
   "validate.publicSshKey": "Must be a valid public key",
+  "validate.yaml.cloud.auth.not.found": "The cloud.yaml must contain an auth key with auth_url, username, and password keys.",
+  "validate.yaml.cloud.cacert.not.found": "If a CA certificate is specified, the cloud auth key must also contain a 'cacert: /etc/openstack-ca/ca.crt' or 'cacert: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem'.",
```

Comment:
I would recomment: `If a CA certificate is specified, the cloud auth key must contain a cacert key with the value '/etc/openstack-ca/ca.crt' or '/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem'.`